### ISSUE TITLE
2016 Primary Election Precinct Results

### ISSUE THREAD

**Issue by dwillis** [openelections/openelections-data-co#34]
-[ ] Adams
-[ ] Alamosa
-[ ] Arapahoe
-[ ] Archuleta
-[ ] Baca
-[ ] Bent
-[ ] Boulder
-[ ] Broomfield
-[ ] Chaffee
-[ ] Cheyenne
-[ ] Clear Creek
-[ ] Conejos
-[ ] Costilla
-[ ] Crowley
-[ ] Custer
-[ ] Delta
-[ ] Denver
-[ ] Dolores
-[ ] Douglas
-[ ] Eagle
-[ ] Elbert
-[ ] El Paso
-[ ] Fremont
-[ ] Garfield
-[ ] Gilpin
-[ ] Grand
-[ ] Gunnison
-[ ] Hinsdale
-[ ] Huerfano
-[ ] Jackson
-[ ] Jefferson
-[ ] Kiowa
-[ ] Kit Carson
-[ ] Lake
-[ ] La Plata
-[ ] Larimer
-[ ] Las Animas
-[ ] Lincoln
-[ ] Logan
-[ ] Mesa
-[ ] Mineral
-[ ] Moffat
-[ ] Montezuma
-[ ] Montrose
-[ ] Morgan
-[ ] Otero
-[ ] Ouray
-[ ] Park
-[ ] Phillips
-[ ] Pitkin
-[ ] Prowers
-[ ] Pueblo
-[ ] Rio Blanco
-[ ] Rio Grande
-[ ] Routt
-[ ] Saguache
-[ ] San Juan
-[ ] San Miguel
-[ ] Sedgwick
-[ ] Summit
-[ ] Teller
-[ ] Washington
-[ ] Weld
-[ ] Yuma

**Comment by amanda-price** [comment]
Working on Arapahoe county!

**Comment by amanda-price** [comment]
Working on El Paso and Routt county!

**Comment by dsharps** [comment]
Working on Adams, Alamosa, Archuleta and Baca counties.

**Comment by amanda-price** [comment]
dsharps, I couldn't find precinct level breakdowns of those counties, could you share where you found the information?

**Comment by dsharps** [comment]
amanda-price, hi! I'm a new volunteer and figuring out the ropes. I was only able to find precinct-level information for the 2008 Alamosa County primary. To summarize, for the 2016 primaries:

Adams: County-level
Alamosa: County-level
Arculeta: County-level
Baca: County-level

Sorry if we duplicated efforts! Maybe I could start from the bottom of the list and work my way up, or let me know if you'd like to divide labor in any way.

**Comment by dwillis** [comment]
@amanda-price @dsharps this is my bad; I should have given Daron more guidance on where to start. I think dividing the counties is a great idea.

**Comment by amanda-price** [comment]
@dwillis @dsharps Yeah that's what I found for the primaries, that a majority of the counties didnt have precinct-level results. I went through the counties to find which one were precinct-level (attached below). There are a few counties where I couldn't find any results online,  but didn't dig into Clarity to see if it was there. It may be that we need to bug some clerks :)

[co_primaryworkbook_2016.xlsx](https://github.com/openelections/openelections-data-co/files/2098735/co_primaryworkbook_2016.xlsx)

**Comment by dsharps** [comment]
@amanda-price Ooooo OK this is super helpful, thank you!! If it's OK with you, I can make phone calls to the clerks of 8 of the 15 counties that don't have results online (will move to the next set of 7 later, unless you beat me to it): Kiowa, Lake, Montezuma, Phillips, Prowers, Sedgwick, Summit, Washington. Sound like a plan?

**Comment by amanda-price** [comment]
@dsharps That sounds great! Definitely let me know how it goes, but that sounds like a great plan!

**Comment by dsharps** [comment]
@amanda-price FYI - I reached out directly to Kiowa, Lake, Montezuma, Phillips, Prowers, Sedgwick, Summit, and Washington counties. Of these, only Lake County had precinct-level results. I've uploaded that data and added source files wherever I could (even if they were at the county level; requested going back a few years if primary or general election data was missing from the source file).

Tomorrow, I'll start working through other counties that don't have results online: Bent, Chaffee, Cheyenne, Costilla, Hinsdale, Huerfano, Jackson!

**Comment by dwillis** [comment]
@dsharps @amanda-price thanks! I think that might be as far as we can get for the 2016 primary, unfortunately. It might be worth grabbing the 2018 primary precinct results if they have them, since it seems like a number of counties don't retain them.
